apps/music/js/ui/views/player_view.js

```diff
@@ -775,10 +777,22 @@ var PlayerView = {
   },
 
   setSeekBar: function pv_setSeekBar(startTime, endTime, currentTime) {
+    if (this.seekBar.max != endTime) {
+      // Duration changed, update accessibility label.
+      navigator.mozL10n.setAttributes(this.seekSlider,
+        'playbackSeekBar', {'duration': formatTime(endTime)});
+    }
+
```

Comment:
In the case above, we creating a human readable label for the seekbar. It changes when seekBar.max changes, which is in this function.